TestData/adf/Release.Notes

```diff
@@ -1 +1 @@

Added additional information on Apple-clicking to the prompt when asking
what files to archive, delete, or add to an archive.

GSHK can now open NuFX archives which are enclosed within MacBinary files.

Added a new preference, "Display warning dialogs" which enables or disables
showing the preceding dialogs.

Added a dialog before making a disk archive which explains what it is, why
you would want to make one and why you -wouldn't- want to make one.  This is
to help out the online services a bit and try to make people -think- before
packing the entire disk.

When creating a file archive if the user has selected "Add Binary II to
this..." I added a dialog which tells the user that their communications
program normally adds Binary II, not GSHK.  This is to lessen the number of
uploads to online services which are double-BNY'd (ie, one Binary II header
added by GSHK and then another by the user's comm program).  Clicking
<Cancel> only backs you up a single level so that you can either uncheck
"Add Binary II" and/or rename the archive.

Added a confirmation dialog before actually deleting files.

Added a confirmation dialog before actually deleting items from an archive.

Added a confirmation dialog before sorting an archive.

Changed the text for the prompt for GSHK & SEA when extracting to say "Extract
the (enclosed) files ("blah" etc.) where?" instead of "Put the files..."
"Extract" better describes what's actually going to happen.

Fixed a problem where archiving from an AppleShare volume resulted in archives
which GSHK and ShrinkIt thought were damaged.

GSHK now strips the high bits off of names of old-style shrinkit disk archives
(archives which were made with Shrinkit 2.1 and earlier have the high bits set
in the disk names).

If you get a CRC error, something should actually be displayed (the dialog 
routine was trying to read a non-existent resource).

Added filename of current filename being extracted to the prompt string.  If
you've got "Prompt on every extraction" on, this is a crude way to find out
what files are actually in the SEA file.

Added "Prompt on every extraction" to SEA wrapper and GSHK core.  This allows
you to prompt for a few files and then turn the prompting off rather than
always have prompting on or always off.  I removed the "Prompt on every
extraction" popup menu from GSHK's preferences dialog.  I hope this is a trend
as that dialog is -way- overloaded.

You can now set "Add Comment to: No Files" and when archiving or adding, GSHK
won't add comments.  This is useful to make sure that no space is wasted when
making SEAs.

Changed routines which deal with dates to use the 5.0.3 (and later) toolbox
call ConvSeconds which does date/time conversion to various formats.  This
saves a few K in the main segment (and leaves me with less code to actually
maintain). The time format is the same time format as in the control panel.

Dave and I converted the GSHK source code to MPW IIGS in one (grueling) weekend
(5/10/92).

```

Comment:
This file is the Release.Notes file extracted from the GS/ShrinkIt distribution (TestData/nufx/GSHK11.SEA). Since it's part of the test data suite it should match the original.